common/transport/amqp/devdoc/amqp_message.md

```diff
@@ -40,31 +40,31 @@ As this is an internal API, the input argument message can be assumed to be of t
 
 **SRS_NODE_IOTHUB_AMQPMSG_05_001: [** The `fromMessage` method shall create a new instance of `AmqpMessage`. **]**
 
-**SRS_NODE_IOTHUB_AMQPMSG_05_002: [** The created `AmqpMessage` object shall have a property of type `Object` named `properties`. **]**
+**SRS_NODE_IOTHUB_AMQPMSG_05_002: [** The created `AmqpMessage` object shall NOT have a property of type `Object` named `properties`. **]**
```

Comment:
could be removed I guess? unless having a properties property makes everything crash?

Reply:
Gone.  But on the plus side now you know.

---
In reply to: [191904474](https://github.com/Azure/azure-iot-sdk-node/pull/300#discussion_r191904474) [](ancestors = 191904474)